// passport-mock-strategy/lib/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var passport_mock_strategy_1 = __importDefault(require("./passport-mock-strategy"));
exports.MockStrategy = passport_mock_strategy_1.default;
exports.Strategy = passport_mock_strategy_1.default;
/**
 * Export everything.
 */
var create_mock_passport_1 = require("./create-mock-passport");
exports.createMockPassport = create_mock_passport_1.default;
var mock_storage_1 = require("./mock-storage");
exports.createMockStorage = mock_storage_1.default;
var mock_user_1 = require("./mock-user");
exports.mockUser = mock_user_1.default;
var mock_utilities_1 = require("./mock-utilities");
exports.connectPassport = mock_utilities_1.connectPassport;
exports.setupSerializeAndDeserialize = mock_utilities_1.setupSerializeAndDeserialize;
exports.default = passport_mock_strategy_1.default;
/**
 * Assign to module.exports for commonjs compatibility
 */
module.exports = passport_mock_strategy_1.default;
Object.assign(module.exports, exports);
